include/ck/tensor_operation/gpu/device/device_grouped_gemm_xdl.hpp

```diff
@@ -17,64 +17,42 @@ namespace ck {
 namespace tensor_operation {
 namespace device {
 
+#define CK_GEMM_DESCS_CONSTANT_BUFF_SIZE 1048576 // 1MB for 1000 gemm_descs
```

Comment:
Better not use marco. You can hardcoded the gemm_descs_const_ sizes, and comment this is constant buffer

Reply:
NVM, I see you are using CK_GEMM_DESCS_CONSTANT_BUFF_SIZE in multiple places